test/src/test/java/com/valqueries/automapper/schema/BaseSchemaBuilderIT.java

```diff
@@ -82,6 +84,34 @@ public void buildSimpleSchema() {
 		});
 	}
 
+	@Test
+	public void buildSimpleSchema_supportTemporal() {
+		builder.addTable(Token.get("TheTable"), tb -> {
+			tb.addColumn(Token.get("id"), UUID.class);
+			tb.addColumn(Token.get("timeField"), ZonedDateTime.class);
+			tb.addPrimaryKey(Token.get("id"));
+		});
+		builder.build();
+
+		ZonedDateTime timeWitlMilliseconds = Instant.ofEpochMilli(1536421864777L).atZone(ZoneOffset.UTC);
+		database.doInTransaction(tx -> {
```

Comment:
It seems that this test is failling in `Mssql`

Reply:
fixed it, the issue was querySingle method which is not working in mssql